go-selinux/selinux_linux.go

```diff
@@ -549,30 +549,30 @@ func (l *level) parseLevel(levelStr string) error {
 
 // rangeStrToMLSRange marshals a string representation of a range.
 func rangeStrToMLSRange(rangeStr string) (*mlsRange, error) {
-	mlsRange := &mlsRange{}
+	mlsrange := &mlsRange{}
```

Comment:
can we just use `range` (or even `r`) here?

Reply:
`range` is a reserved keyboard. Can use `r` if we think it's clear enough